application/controllers/admin/translate.php

```diff
@@ -320,72 +320,6 @@ private function showTranslateAdminmenu($iSurveyID, $survey_title, $tolang)
         return $this->_getLanguageList($iSurveyID, $tolang);
     }
 
-    /*
-    * _getSurveyButton() returns test / execute survey button
-    * @param string $iSurveyID Survey id
-    * @param string $menuitem_url Menu item url
-    */
-    private function _getSurveyButton($iSurveyID, $menuitem_url)
```

Comment:
This function is unused, or?

Reply:
Private function is easy to check if unused : it's unused :)